### DISCUSSION TITLE
Udp playback/ Reverse Buffering

### DISCUSSION THREAD

**Discussion by BILLXZY1215** [TheWidlarzGroup/react-native-video#4953] [Ideas]
### What problem are you trying to solve?

Sometimes mobile devices accept udp/tcp buffers, how can we use these buffers to stream videos?

### Proposed solution

Support UDP playback / reverse buffering to form a video. Can we do reverse buffering to form a video?

### Additional context

Issue [#2254](https://github.com/react-native-video/react-native-video/issues/2254) also mentioned this.

**Comment by freeboub** [comment]
Currently udp is not supported. I know it can be implemented with exoplayer

**Comment by BraveEvidence** [comment]
This will help https://www.youtube.com/watch?v=vVI7ZAZq5e0

**Comment by TWG-AI** [comment]
Thanks for the idea — noted for the team to consider.

> I'm an AI assistant, so double-check any specifics I may have mentioned.
